### PR TITLE
Inverted banner logo on kali.org

### PR DESCRIPTION
**Pull request by ZOASR**

### CODE REVIEW

**Review by Gusted — COMMENTED** [review]
I'm not so sure if this fixes anything.

**Comment by ZOASR** [comment]
> I'm not so sure if this fixes anything.

I did not consider alphapetical order, will fix it!

**Comment by Gusted** [comment]
I actually meant if this fix is necessary as I don't see why this logo should be inverted. As the inverted image has white-ish shadows.

**Comment by ZOASR** [comment]
I just thought the inverted looked alot closer to the original logo, than the default set by the extension.

**Comment by Gusted** [comment]
>  than the default set by the extension

I don't see anything being set by dark reader. I just see the original....

**Comment by ZOASR** [comment]
In Dynamic mode the default looks like this for me, which is not how the logo looks:
image

**Comment by Gusted** [comment]
By any chance on chromium? It seems like I don't have this behavior on Firefox. Let me suggest a fix that won't break on Firefox but will work on Chromium

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -6506,6 +6506,13 @@ IGNORE IMAGE ANALYSIS
 
 ================================
 
+kali.org
+
+INVERT
+#banner-logo
```
Comment (Gusted) [inline src/config/dynamic-theme-fixes.config:6512]:
```suggestion
IGNORE IMAGE ANALYSIS
#banner-logo
```

**Review by Gusted — CHANGES_REQUESTED** [review]

**Comment by ZOASR** [comment]
> By any chance on chromium? 
On chromium yes!
will try it on firefox and see

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (ZOASR) [reply src/config/dynamic-theme-fixes.config:6512]: I tried using `IGNORE IMAGE ANALYSIS` But did not work on chromium

Comment (Gusted) [reply src/config/dynamic-theme-fixes.config:6512]: You need to give it a reload before it takes effect ;)

**Comment by ZOASR** [comment]
It looks fine on firefox, not sure what is the problem!

**Comment by Gusted** [comment]
> It looks fine on firefox, not sure what is the problem!

Don't worry, Firefox is probably missing up some things with the code. I'll have a look later and see what can be done because this logo should be targeted as "light" and be inverted. Even though I agree with you that the original looks better.

The fix I suggested should disable the analysis altogether for that element.

**Review by Gusted — APPROVED** [review]
Thank you!

**Comment by ZOASR** [comment]
> Thank you!

No problem!

**Pull request merged**